cluster/definition.go

```diff
@@ -55,7 +56,7 @@ func WithLegacyVAddrs(feeRecipientAddress, withdrawalAddress string) func(*Defin
 // NewDefinition returns a new definition populated with the latest version, timestamp and UUID.
 // The hashes are also populated accordingly. Note that the hashes need to be recalculated when any field is modified.
 func NewDefinition(name string, numVals int, threshold int, feeRecipientAddresses []string, withdrawalAddresses []string,
-	forkVersionHex string, creator Creator, operators []Operator, random io.Reader, opts ...func(*Definition),
+	forkVersionHex string, creator Creator, operators []Operator, depositAmounts []int, random io.Reader, opts ...func(*Definition),
```

Comment:
Does the fact that we're accepting `int`s here mean no one can make a deposit for e.g 3.5 ether via the CLI? (Probably okay for a CLI restriction once we specify it in the flag description/help text, we can just tell people to craft a definition with the SDK if needs be? (though our createCluster Payload will have to be in Gwei not eth if that is the case))

Reply:
> Does the fact that we're accepting ints here mean no one can make a deposit for e.g 3.5 ether via the CLI?
Correct. To avoid floating points weirdness as we need to be very precise here. We can describe in the documentation the users can leverage SDK when Gweis are the input.